sidestep_engine/core/ema.py

```
@@ -29,10 +29,12 @@ class AdapterEMA:
             Typical: 0.9999 (slow), 0.999 (faster tracking).
     """
 
-    def __init__(self, params: Iterable[torch.nn.Parameter], decay: float) -> None:
+    def __init__(self, params: Iterable[torch.nn.Parameter], decay: float, start_step: int = 2000) -> None:
         if not (0.0 <= decay < 1.0):
             raise ValueError(f"EMA decay must be >= 0 and < 1 (got {decay})")
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)